### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Haywhizzz**
In this project I have

Webpack:

Imported CSS file in index.js file.
Separated files for each page.
Compiled all .js files into index.js

DOM:

Created a Home page in home.js and load it when opening the page.
Created a menu page and load it when clicking on the menu link.
Created a contact page and load it when clicking on the contact link.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Haywhizzz 👋

**You've done pretty well here.**

However, we are a step from moving on.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "A simple restaurant website made with ES6 using Webpack, to showcase DOM manipulation by dynamically generating the entire contents of the restaurant homepage using JavaScript",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (mikenath223) [inline package.json]: - [x] Kindly add the neccessary start scripts:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

Comment (mikenath223) [reply package.json]: - Also if the optional start script is used make sure to include the `web-dev-server` package.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "A simple restaurant website made with ES6 using Webpack, to showcase DOM manipulation by dynamically generating the entire contents of the restaurant homepage using JavaScript",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack -p",
+    "dev": "webpack-dev-server --open"
```
Comment (whiz25) [inline package.json]: - Please use the same commands as here. Pay attention to the `"dev"` script
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🤞🏿 
You did well in this PR @Haywhizzz . Your README is professional and the app looks good However, the `scripts` section of the `pakage.json` files needs fixing. See my comment.


### Martin Afani

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Great job @team, thanks for making the required changes.

now you can merge, your project is approved.

## Happy Coding :tophat:

**Pull request merged**